examples/supertuxkart/Dockerfile

```diff
@@ -16,45 +16,25 @@
 FROM golang:1.17.2 as go-builder
 WORKDIR /go/src/supertuxkart
 
-RUN apt-get update && apt-get install -y curl tar xz-utils
-
 COPY main.go .
 COPY go.mod .
 RUN go mod tidy
 RUN go build -o wrapper .
 
 # compile stk
-FROM debian:buster as stk-builder
-
-RUN apt update && apt-get install -y build-essential cmake libbluetooth-dev \
-libcurl4-openssl-dev libenet-dev libfreetype6-dev libharfbuzz-dev libfribidi-dev \
-libgl1-mesa-dev libglew-dev libjpeg-dev libogg-dev libopenal-dev libpng-dev \
-libssl-dev libvorbis-dev libxrandr-dev libx11-dev nettle-dev pkg-config zlib1g-dev \
-git subversion
+FROM debian:bullseye
```

Comment:
Line above should probably be something like "Download STK" rather than "compile", since it's no longer compiling.